### ISSUE TITLE
If `eslint.config.js` is available 

### ISSUE THREAD

**Issue by jkroepke** [super-linter/super-linter#6448]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using the latest Super-linter version available?

- [X] I am using the latest Super-linter version.
- [X] I can reproduce the issue running Super-linter using complete version identifier (example: vX.Y.Z), and not just with a partial one (example: vX)

- [X] I am using the super-linter/super-linter action or container image, and not the deprecated github/super-linter action or container image.

### Are you resonably sure that it's a Super-linter issue, and not an issue related to a tool that Super-linter runs?

- [X] I think that this is a Super-linter issue.

### Current Behavior

My repository contains a `eslint.config.js` and I would like to use for super-linter as well. However, it seems like super-linter json linter errored with the following error:

```
  Invalid option '--ext' - perhaps you meant '-c'?
  You're using eslint.config.js, some command line flags are no longer available. Please see https://eslint.org/docs/latest/use/command-line-interface for details.
```

### Expected Behavior

super-linter should detect `eslint.config.js` and if present, legacy CLI options should not be set.

### Super-Linter version

```markdown
v7.2.1
```

### Super-linter configuration

```yaml
steps:
      - name: Checkout code
        uses: actions/checkout@v4
        with:
          # super-linter needs the full git history to get the
          # list of files that changed across commits
          fetch-depth: 0

      - name: Super-linter
        uses: super-linter/super-linter@v7.2.1 # x-release-please-version
        env:
          DEFAULT_BRANCH: origin/main
          EDITORCONFIG_FILE_NAME: .editorconfig-checker.json
          FILTER_REGEX_EXCLUDE: >-
            (CHANGELOG-pre-.+.md|.*drawio|local-testing/|hack/|charts/(opsstack-crds|opsstack/templates/|opsstack/charts/)|terraform/tf-mod-lib)
          # To report GitHub Actions status checks
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          LINTER_RULES_PATH: .
          MULTI_STATUS: false
          VALIDATE_ANSIBLE: false
          VALIDATE_ARM: false
          VALIDATE_CPP: false
          VALIDATE_CHECKOV: false
          VALIDATE_CLANG_FORMAT: false
          VALIDATE_CLOJURE: false
          VALIDATE_CLOUDFORMATION: false
          VALIDATE_COFFEESCRIPT: false
          VALIDATE_CSHARP: false
          VALIDATE_CSS: false
          VALIDATE_CSS_PRETTIER: false
          VALIDATE_DART: false
          VALIDATE_DOCKERFILE_HADOLINT: false
          VALIDATE_DOTNET_SLN_FORMAT_ANALYZERS: false
          VALIDATE_DOTNET_SLN_FORMAT_STYLE: false
          VALIDATE_DOTNET_SLN_FORMAT_WHITESPACE: false
          VALIDATE_GHERKIN: false
          VALIDATE_GIT_COMMITLINT: false
          VALIDATE_GITLEAKS: false
          VALIDATE_GO: false
          VALIDATE_GO_MODULES: false
          VALIDATE_GO_RELEASER: false
          VALIDATE_GOOGLE_JAVA_FORMAT: false
          VALIDATE_GROOVY: false
          VALIDATE_HTML: false
          VALIDATE_HTML_PRETTIER: false
          VALIDATE_JAVA: false
          VALIDATE_JAVASCRIPT_ES: false
          VALIDATE_JAVASCRIPT_PRETTIER: false
          VALIDATE_JAVASCRIPT_STANDARD: false
          VALIDATE_JSCPD: false
          VALIDATE_JSON_PRETTIER: false
          VALIDATE_JSONC: false
          VALIDATE_JSONC_PRETTIER: false
          VALIDATE_JSX: false
          VALIDATE_JSX_PRETTIER: false
          VALIDATE_KOTLIN: false
          VALIDATE_KUBERNETES_KUBECONFORM: false
          VALIDATE_LATEX: false
          VALIDATE_LUA: false
          VALIDATE_MARKDOWN_PRETTIER: false
          VALIDATE_OPENAPI: false
          VALIDATE_PERL: false
          VALIDATE_PHP: false
          VALIDATE_PHP_BUILTIN: false
          VALIDATE_PHP_PHPCS: false
          VALIDATE_PHP_PHPSTAN: false
          VALIDATE_PHP_PSALM: false
          VALIDATE_PROTOBUF: false
          VALIDATE_R: false
          VALIDATE_RAKU: false
          VALIDATE_RUBY: false
          VALIDATE_RUST_2015: false
          VALIDATE_RUST_2018: false
          VALIDATE_RUST_2021: false
          VALIDATE_RUST_CLIPPY: false
          VALIDATE_SCALAFMT: false
          VALIDATE_SNAKEMAKE_LINT: false
          VALIDATE_SNAKEMAKE_SNAKEFMT: false
          VALIDATE_STATES: false
          VALIDATE_SQLFLUFF: false
          VALIDATE_TEKTON: false
          VALIDATE_TERRAFORM_TERRASCAN: false
          VALIDATE_TSX: false
          VALIDATE_TYPESCRIPT_ES: false
          VALIDATE_TYPESCRIPT_PRETTIER: false
          VALIDATE_TYPESCRIPT_STANDARD: false
          VALIDATE_VUE_PRETTIER: false
          VALIDATE_XML: false
          VALIDATE_YAML_PRETTIER: false
          YAML_CONFIG_FILE: .yamllint.yaml
          YAML_ERROR_ON_WARNING: true
```

### Relevant log output

```shell
2024-12-17 08:29:17 [INFO]   Linting JSON items...
  Error: -17 08:29:18 [ERROR]   Found errors when linting JSON. Exit code: 1.
  2024-12-17 08:29:18 [INFO]   Stderr contents for JSON:
  ------
  Invalid option '--ext' - perhaps you meant '-c'?
  You're using eslint.config.js, some command line flags are no longer available. Please see https://eslint.org/docs/latest/use/command-line-interface for details.
```

### Steps To Reproduce

1. configure eslint using `eslint.config.js`
2. run super-linter

**Comment by Falcion** [comment]
I think this issue is part of my issue #6405.

Currently super-linter doesn't support v9 of ESLint, only v8 (which means no `eslint.config.js` and etc.).

**Comment by ferrarimarco** [comment]
Hi! This is indeed a duplicate of #6405.
